controllers/operands/kubevirt_test.go

```diff
@@ -2524,6 +2524,70 @@ Version: 1.2.3`)
 			}))
 		})
 
+		Context("Tune KubeVirt rateLimiters", func() {
+
+			var hco *hcov1beta1.HyperConverged
+
+			BeforeEach(func() {
+				hco = commonTestUtils.NewHco()
+			})
+			It("Should be empty by default", func() {
+				kv, err := NewKubeVirt(hco)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(hco.Spec.TuningPolicy).To(BeEmpty())
+				Expect(kv.Spec.Configuration.APIConfiguration).To(BeNil())
+				Expect(kv.Spec.Configuration.ControllerConfiguration).To(BeNil())
+				Expect(kv.Spec.Configuration.WebhookConfiguration).To(BeNil())
+				Expect(kv.Spec.Configuration.HandlerConfiguration).To(BeNil())
+
+			})
+
+			It("Should be empty if annotation is not present", func() {
+				hco.Spec.TuningPolicy = hcov1beta1.HyperConvergedAnnotationTuningPolicy
+
+				kv, err := NewKubeVirt(hco)
+
+				Expect(err).To(HaveOccurred())
+				Expect(err).Should(MatchError("tuning policy set but annotation not present or wrong"))
```

Comment:
Not sure about this error. It does make sense though. Please remember to well document this behaviour.

Reply:
Sure!